examples/platformer/main.go

```diff
@@ -0,0 +1,90 @@
+package main
+
```

Comment:
Add the build tag comment:

`// +build example`

This is useful to prevent users from installing examples with `go get github.com/hajimehoshi/ebiten/...`